Makefile

```diff
@@ -37,15 +44,20 @@ embedmd:
 docs: embedmd
 	embedmd -w `find Documentation -name "*.md"`
 
-assets: gobindata
+bindata: pkg/manifests/bindata.go
```

Comment:
How about getting rid of the `bindata` target and just adding `pkg/manifests/bindata.go` as a dependency to `build`?

Reply:
previously we had a phony target that was used as a shortcut for the `generate` target. this preserves that functionality while using a better name

Reply:
also, for better locality, I think it's preferable to have a single reference to external files so that updating a file name means we don't have to chase down multiple references everywhere, but one single reference. This meas either using another variable or a phony target.

Reply:
talked offline with @mxinden. we will do a followup PR to further cleanup the Makefile